kuma/core/pipeline/sass.py

```diff
@@ -0,0 +1,13 @@
+from pipeline.compilers import SubProcessCompiler
+
+
+class DebugSassCompiler(SubProcessCompiler):
+    output_extension = 'css'
+
+    def match_file(self, filename):
+        return filename.endswith(('.scss', '.sass'))
+
+    def compile_file(self, infile, outfile, outdated=False, force=False):
+        # as gulp is compiling .scss files when DEBUG=True, all we need to do
+        # here is give Pipeline the location of the pre-compiled .css file
+        return outfile
```

Comment:
The output of the ``compile_file`` method is not used, so there is no need to return the ``outfile`` here. In fact, since we want ``DebugSassCompiler`` to act the same as ``pipeline.compilers.sass.SASSCompiler`` except that we don't want it to compile anything, I'd recommend this:
```
from pipeline.compilers.sass import SASSCompiler


class DebugSassCompiler(SASSCompiler):
    """
    This class is a drop-in replacement for SASSCompiler, and
    behaves like that class in all aspects except compilation.
    """

    def compile_file(self, *args, **kwargs):
        return
```
so we track any potential future changes to ``pipeline.compilers.sass.SASSCompiler``.